src/main/java/hudson/plugins/spotinst/cloud/BaseSpotinstCloud.java

```diff
@@ -587,25 +591,31 @@ protected Integer getNumOfExecutors(String instanceType) {
         Integer retVal;
 
         Integer globalOverrideExecutorsNumber = getExecutorsFromGlobalOverride();
+        boolean isOneOffNodesEnabled          = getIsOneOffNodesEnabled();
```

Comment:
getExecutorsFromGlobalOverride => should be inside the else anyway

Reply:
done